sequencer/src/persistence/sql.rs

```diff
@@ -1757,52 +2256,55 @@ mod test {
             .proposal
             .justify_qc
             .data
-            .leaf_commit = Committable::commit(&leaf.clone().into());
+            .leaf_commit = Committable::commit(&leaf.clone());
         let qc = next_quorum_proposal.data.justify_qc();
 
         // Add to database.
         storage
-            .append_da(&da_proposal, VidCommitment::V0(payload_commitment))
+            .append_da2(&da_proposal, VidCommitment::V1(payload_commitment))
             .await
             .unwrap();
-        storage.append_vid(&vid_share).await.unwrap();
         storage
-            .append_quorum_proposal(&quorum_proposal)
+            .append_vid2(&convert_proposal(vid_share.clone()))
+            .await
+            .unwrap();
+        storage
+            .append_quorum_proposal2(&quorum_proposal)
             .await
             .unwrap();
 
         // Add an extra quorum proposal so we have a QC pointing back at `leaf`.
         storage
-            .append_quorum_proposal(&next_quorum_proposal)
+            .append_quorum_proposal2(&next_quorum_proposal)
             .await
             .unwrap();
 
         // Fetch it as if we were rebuilding an archive.
+        // TODO (abdul):
```

Comment:
@imabdulbasit another TODO comment

Reply:
related to vid common field. I'll remove it